app/javascript/dashboard/routes/dashboard/contacts/components/NoteList.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <div class="notelist-wrap">
+      <h3 class="block-title">
+        Notes
+      </h3>
+      <add-note @add="onAddNote" />
+      <contact-note
+        v-for="note in notes"
+        :key="note.id"
+        :note="note.content"
+        :user-name="note.user.name"
+        :time-stamp="note.created_at"
+        :thumbnail="note.user.thumbnail"
+        @edit="onRemoveNote"
+        @delete="onDeleteNote"
+      />
+      <div class="button-wrap">
+        <woot-button variant="clear link" class="button" @click="onclick">
+          View all notes
+          <i class="ion-arrow-right-c" />
+        </woot-button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import ContactNote from './ContactNote';
+import AddNote from './AddNote';
+
+export default {
+  components: {
+    ContactNote,
+    AddNote,
+  },
+
+  props: {
+    notes: {
+      type: Array,
+      default: () => [],
+    },
+  },
+
+  methods: {
+    onclick() {
+      this.$emit('show');
+    },
+    onAddNote() {
+      this.$emit('addNote');
```

Comment:
All the below events needs to show the data passed from child components. Like for edit and delete `id` of the note,  on add `content` of the note, these needs to passed.